staging/src/k8s.io/api/flowcontrol/v1beta3/types.go

```diff
@@ -103,10 +103,25 @@ const (
 	AutoUpdateAnnotationKey = "apf.kubernetes.io/autoupdate-spec"
 )
 
+const (
+	// This annotation applies to v1beta3 only.
+	//
+	// The presence of this annotation in a v1beta3 object means that
+	// a zero value in the 'NominalConcurrencyShares' field means zero
+	// rather than the old default of 30.
+	//
+	// To set a zero value for the 'NominalConcurrencyShares' field,
+	// set the annotation to an empty string value:
+	//   "flowcontrol.k8s.io/concurrency-shares-defaults-to-zero": ""
```

Comment:
This is more restrictive than the plan we are following. Is that because you are applying Postel's Law here?

Reply:
This is documentation about more than what is produced in that scenario, it also has to describe what is consumed in other scenarios.